straxen/mini_analysis.py

```diff
@@ -39,7 +39,8 @@ def wrapped_f(context: strax.Context, run_id: str, **kwargs):
             known_kwargs = (
                 "time_range seconds_range time_within time_selection "
                 "ignore_time_warning "
-                "selection_str t_reference to_pe config"
+                "selection_str t_reference to_pe config "
+                "corrections_run_id"
```

Comment:
You can add another argument to `mini_analysis` function so that it can add additional `known_kwargs` to the default ones.